spec/lib/iiif_manifest/v3/manifest_factory_spec.rb

```diff
@@ -204,7 +204,7 @@ def display_content
 
       it 'does not have a rendering on the sequence' do
         allow(book_presenter).to receive(:file_set_presenters).and_return([file_presenter])
-        expect(result['rendering']).to eq []
+        expect(result['rendering']).to be_nil
```

Comment:
Do you think it would be better to check that the `rendering` key doesn't exist rather than it's value is nil?
```suggestion
        expect(result.key?('rendering')).to be_false
```

Reply:
Ah yes, I agree